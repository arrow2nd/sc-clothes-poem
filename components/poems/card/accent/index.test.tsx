import { render } from '@testing-library/react'

import Accent from './index'

test('Accent', () => {
  const { container } = render(<Accent bgColor="F54275" />)
  expect(container).toMatchSnapshot()
})
