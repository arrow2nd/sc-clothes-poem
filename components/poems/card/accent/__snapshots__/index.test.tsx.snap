// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accent 1`] = `
<div>
  <div
    class="mb-4 py-0.5 w-10 rounded-full border border-gray-100"
    style="background-color: rgb(245, 66, 117);"
  />
</div>
`;
